website/docs/reference/global-configs/command-line-options.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Command line flags"
-id: "command-line-flags"
-sidebar: "Command line flags"
+title: "Command line options"
+id: "command-line-options"
+sidebar: "Command line options"
```

Comment:
Should we change the use of `flags` in the body of this doc to `options`?

Reply:
["Flags"](https://docs-getdbt-com-git-jerco-flags-behavior-changes-dbt-labs.vercel.app/reference/global-configs/about-global-configs) is the unifying description we're giving, so the body of this doc can stay as-is.

### [More detail](https://docs-getdbt-com-git-jerco-flags-behavior-changes-dbt-labs.vercel.app/reference/global-configs/about-global-configs#setting-flags)

There are multiple ways of setting flags, which depend on the use case:

- **[Project-level `flags` (`dbt_project.yml`)](https://docs-getdbt-com-git-jerco-flags-behavior-changes-dbt-labs.vercel.app/reference/global-configs/project-flags):** Define version-controlled defaults for everyone running this project. Preserve legacy behaviors until their slated deprecation.
- **[Environment variables](https://docs-getdbt-com-git-jerco-flags-behavior-changes-dbt-labs.vercel.app/reference/global-configs/environment-variable-configs):** Define different behavior in different runtime environments (development vs. production vs. [continuous integration](https://docs-getdbt-com-git-jerco-flags-behavior-changes-dbt-labs.vercel.app/docs/deploy/continuous-integration), or different behavior for different users in development (based on personal preferences).
- **[CLI options](https://docs-getdbt-com-git-jerco-flags-behavior-changes-dbt-labs.vercel.app/reference/global-configs/command-line-options):** Define behavior specific to _this invocation_. Supported for all dbt commands.